test/e2e/commands/node-update.test.mjs

```diff
@@ -0,0 +1,130 @@
+import { flags } from '../../../src/commands/index.mjs'
+import { constants } from '../../../src/core/index.mjs'
+import {
+  accountCreationShouldSucceed,
+  balanceQueryShouldSucceed,
+  bootstrapNetwork,
+  getDefaultArgv, getNodeIdsPrivateKeysHash, getTmpDir,
+  HEDERA_PLATFORM_VERSION_TAG
+} from '../../test_util.js'
+import { getNodeLogs } from '../../../src/core/helpers.mjs'
+import { NodeCommand } from '../../../src/commands/node.mjs'
+import { HEDERA_HAPI_PATH, ROOT_CONTAINER } from '../../../src/core/constants.mjs'
+import fs from 'fs'
+
+describe('Node add', () => {
+  const defaultTimeout = 120000
+  const namespace = 'node-update'
+  const updateNodeId = 'node2'
+  const newAccountId = '0.0.7'
+  const argv = getDefaultArgv()
+  argv[flags.keyFormat.name] = constants.KEY_FORMAT_PEM
+  argv[flags.nodeIDs.name] = 'node1,node2,node3'
+  argv[flags.nodeID.name] = updateNodeId
+
+  argv[flags.newAccountNumber.name] = newAccountId
+  argv[flags.newAdminKey.name] = '302e020100300506032b6570042204200cde8d512569610f184b8b399e91e46899805c6171f7c2b8666d2a417bcc66c2'
+
+  argv[flags.generateGossipKeys.name] = true
+  argv[flags.generateTlsKeys.name] = true
+  argv[flags.keyFormat.name] = constants.KEY_FORMAT_PEM
+  // set the env variable SOLO_FST_CHARTS_DIR if developer wants to use local FST charts
+  argv[flags.chartDirectory.name] = process.env.SOLO_FST_CHARTS_DIR ? process.env.SOLO_FST_CHARTS_DIR : undefined
+  argv[flags.releaseTag.name] = HEDERA_PLATFORM_VERSION_TAG
+  argv[flags.namespace.name] = namespace
+  argv[flags.persistentVolumeClaims.name] = true
+  const bootstrapResp = bootstrapNetwork(namespace, argv)
+  const nodeCmd = bootstrapResp.cmd.nodeCmd
+  const accountCmd = bootstrapResp.cmd.accountCmd
+  const k8 = bootstrapResp.opts.k8
+  let existingServiceMap
+  let existingNodeIdsPrivateKeysHash
+
+  afterAll(async () => {
+    await getNodeLogs(k8, namespace)
+    await k8.deleteNamespace(namespace)
+  }, 600000)
+
+  it('cache current version of private keys', async () => {
+    existingServiceMap = await nodeCmd.accountManager.getNodeServiceMap(namespace)
+    existingNodeIdsPrivateKeysHash = await getNodeIdsPrivateKeysHash(existingServiceMap, namespace, constants.KEY_FORMAT_PEM, k8, getTmpDir())
+  }, defaultTimeout)
+
+  it('should succeed with init command', async () => {
+    const status = await accountCmd.init(argv)
+    expect(status).toBeTruthy()
+  }, 450000)
+
+  it('should update a new node property successfully', async () => {
+    // generate gossip and tls keys for the updated node
+    const tmpDir = getTmpDir()
+    const signingKey = await nodeCmd.keyManager.generateSigningKey(updateNodeId)
+    const signingKeyFiles = await nodeCmd.keyManager.storeSigningKey(updateNodeId, signingKey, tmpDir)
+    nodeCmd.logger.debug(`generated test gossip signing keys for node ${updateNodeId} : ${signingKeyFiles.certificateFile}`)
+    argv[flags.gossipKey.name] = signingKeyFiles.certificateFile
```

Comment:
```suggestion
    argv[flags.gossipKey.name] = signingKeyFiles.privateKeyFile
```

Reply:
Updated with separate flags for public and private key